config/routes.rb

```diff
@@ -2,11 +2,7 @@
   root to: "pages#home"
 
   # post '/linter', to: "linter#new"
-  resources :ruleset, only: [:new, :create]
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :ruleset, only: %i[new create index]
```

Comment:
Using a singular with `resources` causes some strangeness with the index and create path helpers. If this line is changed to:
```ruby
resources :rulesets, only: %i[new create index]
```
The path helper will be:
- new `new_ruleset_path`
- create `rules_sets_path`
- index `rules_sets_path`